karton/system/system.py

```diff
@@ -66,39 +66,39 @@ def gc_collect_resources(self) -> None:
         if resources_to_remove:
             self.backend.remove_objects(karton_bucket, resources_to_remove)
 
-    def gc_collect_abandoned_queues(self):
-        online_consumers = self.backend.get_online_consumers()
-        for bind in self.backend.get_binds():
-            identity = bind.identity
-            if identity not in online_consumers and not bind.persistent:
-                # If offline and not persistent: remove queue
-                for queue in self.backend.get_queue_names(identity):
-                    self.log.info(
-                        "Non-persistent: unwinding tasks from queue %s", queue
-                    )
-                    removed_tasks = self.backend.remove_task_queue(queue)
-                    for removed_task in removed_tasks:
-                        self.log.info("Unwinding task %s", str(removed_task.uid))
-                        # Mark task as finished
-                        self.backend.set_task_status(removed_task, TaskState.FINISHED)
-                    self.log.info("Non-persistent: removing bind %s", identity)
-                    self.backend.unregister_bind(identity)
-
     def gc_collect_tasks(self) -> None:
         # Collects finished tasks
         root_tasks = set()
         running_root_tasks = set()
-        tasks = self.backend.get_all_tasks()
-        enqueued_task_uids = self.backend.get_task_ids_from_queue(KARTON_TASKS_QUEUE)
+        unrouted_task_fquids = self.backend.get_task_ids_from_queue(KARTON_TASKS_QUEUE)
 
         current_time = time.time()
         to_delete = []
 
-        for task in tasks:
+        queues_to_remove = set()
```

Comment:
Can't decide between `to_remove` and `to_clear`. While the queues are removed, this list is then used to clear the queues